Remove aggregate special stone recipes

CivTopWolf opened the pull request:
The special stones, Andesite, Granite, and Diorite, are already abundant on the server. They are not available in all biomes, according to @CaptainKlutz due to a config error, however regardless of the cause this does not cause issues, and instead actually makes the game more interesting in the spirit of RealisticBiomes- people who don't live near mesa or savanna biomes will need to trade with those that do. Markets for special stone already exist in the two largest cities, Mount Augusta and Yoahtl, addressing this need organically, and adding this recipe hurts the people who invested time solving this shortage in-game.

One year into the server being alive, we should not be adding config recipes for resources that are already possible to obtain in-game, and instead should only add recipes for those few that can't be.

CivTopWolf committed:
Remove aggregate special stone recipes

The special stones, Andesite, Granite, and Diorite, are already abundant on the server. They are not available in all biomes, according to @CaptainKlutz due to a config error, however regardless of the cause this does not cause issues, and instead actually makes the game more interesting in the spirit of RealisticBiomes- people who don't live near mesa or savanna biomes will need to trade with those that do. Markets for special stone already exist in the two largest cities, Mount Augusta and Yoahtl, addressing this need organically, and adding this recipe hurts the people who invested time solving this shortage in-game. One year into the server being alive, we should not be adding config recipes for resources that are already possible to obtain in-game, and instead should only add recipes for those few that can't be.

CaptainKlutz commented:
I agree with this change only of the 3 proposed (I argue to keep the purpur because it is a vanilla recipe and does not make it any more plentiful, and keep the red netherbrick because otherwise there's an inconsistency with normal netherbrick). My apologies to all involved; next time I will check with the community before writing PRs.

@ribagi could you please merge this request when you can? The community at large is opposed to it. Sorry for all the fuss.

ribagi commented:
@CaptainKlutz Defend your shit. Don't just retreat from your position. If you asked me to accept a pull request, don't just back away from the smallest amount of backlash. 

@CivTopWolf I rather this to be a small increase of efficiency for players rather than a full revert.  The default MC recipe is 1 cobblestone + 1 quartz = 1 Diorite. The one Klutz wrote is 2 cobblestone + 1 quartz = 2 Diorite. If that doesn't work make it 4 cobblestone + 3 quartz = 4 Diorite. I am not simply going to remove something because someone is making a marginal amount of diamonds.

CivTopWolf commented:
@ribagi, thanks for responding.

1. Much of the community supports my position including sashimii (founder of CW), and bgbba (Yoahtl). [My post on the topic received 25 upvotes.](https://www.reddit.com/r/civclassics/comments/8nap7p/why_resource_scarcity_matters/) They don't have any stake in the issue. That post alone has 115 comments and inspired multiple other posts. What is your bar for a non-small amount of backlash?
2. I'm not sure if by "I am not simply going to remove something because someone is making a marginal amount of diamonds." you're emphasis is on "someone"-- actually people in every major market, Shopside, MTA, and Yoahtl, trade each of the special stones.
3. Given that multiple people trade special stones, these recipe devalues the time we spent gathering those to supply the market. I find this an misuse of admin powers no different than directly removing diamonds from the personal chests of multiple players.
4. While I do personally trade these special stones, I am not submitted this PR to enrich myself. I am much more concerned about the precedent that this sets. Anything that decreases resource scarcity hurts trade. If anyone can submit a PR that arbitrarily makes thing cheaper, why not add recipes to generate sand out of cobble? Or ink sacs out of cobble? Or anything else that mildly inconveniences anyone? Must all aesthetic blocks be convenient to obtain? Is there no interest in having "luxury" goods? You'll notice in addition to this PR, I also submitted PRs to remove the other recipes that CaptainKlutz recently added (which I have no personal stake in).
5. [Here is my complete argument for why factory recipes like this hurt trade.](https://www.reddit.com/r/civeconomics/comments/8ngrao/how_factory_recipes_hurt_trade/) I explained in another post [why making resources abundant hurts trade, which in turn hurts the server](https://www.reddit.com/r/civclassics/comments/8nap7p/why_resource_scarcity_matters/).

CivTopWolf committed:
Make special stone recipes

Set the efficiency of special stone recipes to that of their vanilla counter parts.

https://minecraft.gamepedia.com/Andesite (1 Cobble + 1 Diorite : 1 Andesite)
https://minecraft.gamepedia.com/Diorite  (1 Cobble + 1 Nether Quartz : 1 Diorite)
https://minecraft.gamepedia.com/Granite (1 Diorite + 1 Nether Quartz : 1 Granite)

This reduces toil for players who wish to produce these in big batches.
